.golangci.yml

```diff
@@ -20,7 +20,7 @@ linters:
     - gosimple
     - govet
     - ineffassign
-    - interfacer
+    # - interfacer
```

Comment:
why this was commented?

Reply:
We decided to disable it (conversation started here: https://github.com/tendermint/tendermint/pull/5219#discussion_r467780016)